web/sites/default/config/webform.webform.report_campsite.yml

```diff
@@ -737,15 +744,15 @@ handlers:
       requester_name: contact_name
       requester_email: contact_email
       subject: 'Campsite Report Inside Park'
-      comment: "A campsite has been reported inside of a park by a community member:\r\n\r\n[webform_submission:values:html]"
+      comment: "A campsite has been reported inside of a park by a community member:\r\n\r\n[webform_submission:values:html]\r\n\r\n<br><strong>Related campsite report:</strong>\r\n[webform_submission:values:report_ticket_id]"
       tags: 'drupal webform campsite'
       priority: normal
       status: new
       group_id: '4549352062487'
       assignee_id: ''
       type: incident
       collaborators: ''
-      custom_fields: "6353388345367: 'report_campsite'\r\n5581480390679: ['[webform_submission:values:report_location:location_lat]', 'report_location:location_lat']\r\n5581490332439: ['[webform_submission:values:report_location:location_lon]', 'report_location:location_lon']\r\n13407901552407: ['[webform_submission:values:report_location:location_x]', 'report_location:location_x']\r\n13407918835095: ['[webform_submission:values:report_location:location_y]', 'report_location:location_y']\r\n1500012743961: ['[webform_submission:values:report_location:location_address]', 'report_location:location_address']\r\n1500013095781: '[webform_submission:uuid]'\r\n9205221816983: ['[webform_submission:values:computed_property_type]', 'property_type']\r\n5873270225303: '[webform_submission:values:report_visible_items:checked:vehicle]'"
+      custom_fields: "6353388345367: 'report_campsite'\r\n5581480390679: ['[webform_submission:values:report_location:location_lat]', 'report_location:location_lat']\r\n5581490332439: ['[webform_submission:values:report_location:location_lon]', 'report_location:location_lon']\r\n13407901552407: ['[webform_submission:values:report_location:location_x]', 'report_location:location_x']\r\n13407918835095: ['[webform_submission:values:report_location:location_y]', 'report_location:location_y']\r\n1500012743961: ['[webform_submission:values:report_location:location_address]', 'report_location:location_address']\r\n1500013095781: '[webform_submission:uuid]'\r\n9205221816983: ['[webform_submission:values:computed_property_type]', 'property_type']\r\n5873270225303: '[webform_submission:values:report_visible_items:checked:vehicle]'\r\n8692493739543: '[webform_submission:values:report_ticket_id]'"
```

Comment:
Stores the related campsite ticket ID in the ticket created for park rangers, in the Related Ticket ID custom field in Zendesk. This custom field already exists but is not being added to the campsite form, so this does not change the schema. This handler is currently disabled.